fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2537,8 +2537,19 @@ class FMCMainDisplay extends BaseAirliners {
     }
 
     getOrSelectILSsByIdent(ident, callback) {
-        this._getOrSelectWaypoints(this.dataManager.GetILSsByIdent.bind(this.dataManager), ident, callback); // TODO port over (fms-v2)
+        this._getOrSelectWaypoints(
+            async () => this.navigationDatabase.searchVor(ident).then((vors) => vors.filter((vor) => vor.type & 1 << 7 /* ILS */)),
+            ident,
+            (selectedVor) => {
+                const airportIdent = selectedVor.databaseId.slice(3, 7).trim()
```

Comment:
Please do not use databaseId like this. The format of it varies depending on the database backend!